### PR TITLE
Add customizePage method to imageSnapshot

### PR DESCRIPTION
**Pull request by igor-dv**
Issue: There is no ability to customize the `page` instance before it goes to a page.

## What I did
I've added a `customizePage` method that will be invoked before page calls to `goto`.

### CODE REVIEW

File: addons/storyshots/storyshots-puppeteer/README.md [hunk 1]
```diff
@@ -114,6 +115,32 @@ const chromeExecutablePath = '/usr/local/bin/chrome';
 initStoryshots({suite: 'Image storyshots', test: imageSnapshot({storybookUrl: 'http://localhost:6006', chromeExecutablePath})});
 ```
 
+### Customizing a `page` instance
+
+Sometimes, there is a need to customize a page before it calls to the `goto` api. 
+
+Here is an example for a device emulation:
```
Comment (wuweiweiwu) [inline addons/storyshots/storyshots-puppeteer/README.md]: suggestion:

> an example of device emulation

File: addons/storyshots/storyshots-puppeteer/README.md [hunk 2]
```diff
@@ -114,6 +115,32 @@ const chromeExecutablePath = '/usr/local/bin/chrome';
 initStoryshots({suite: 'Image storyshots', test: imageSnapshot({storybookUrl: 'http://localhost:6006', chromeExecutablePath})});
 ```
 
+### Customizing a `page` instance
+
+Sometimes, there is a need to customize a page before it calls to the `goto` api.
```
Comment (wuweiweiwu) [inline addons/storyshots/storyshots-puppeteer/README.md]: suggestion:

> before it calls the `goto` api

File: addons/storyshots/storyshots-puppeteer/README.md [hunk 3]
```diff
@@ -151,4 +178,4 @@ If you run your test without either the static build or a running instance, this
 
 To make sure your screenshots are taken from latest changes of your Storybook, you must keep your static build or running Storybook up-to-date.
 This can be achieved by adding a step before running the test ie: `yarn run build-storybook && yarn run image-snapshots`.
-If you run the image snapshots against a running Storybook in dev mode, you don't have to care about being up-to-date because the dev-server is watching changes and rebuilds automatically.
\ No newline at end of file
+If you run the image snapshots against a running Storybook in dev mode, you don't have to care about being up-to-date because the dev-server is watching changes and rebuilds automatically.
```
Comment (wuweiweiwu) [inline addons/storyshots/storyshots-puppeteer/README.md]: suggestion: 

> don't have to worry about the snapshots being up-to-date

**Review by wuweiweiwu — APPROVED** [review]
Looks awesome! Just some minor wording suggestions. LGTM 👍

**Comment by igor-dv** [comment]
@wuweiweiwu thanks for the review =)

**Pull request merged**